planutils/packages/pyperplan/install

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+pip3 install pyperplan
```

Comment:
It's recommended to use `python3 -m pip` instead of `pip3`, same for the uninstall script below.